### ISSUE TITLE
when training  “the loss_d_real is negative value ” is OK？ and why

### ISSUE THREAD

**Issue by talengu** [albertpumarola/GANimation#114]
The result is amazing! I have two questions about loss.

1. But I have a question that the three loss below will influence each other？some positive some negative.

“self._loss_d_real + self._loss_d_cond + self._loss_d_fake”

2. when I am training  “loss_d_real is negative value” is right" and why
```python
self._loss_d_real = self._compute_loss_D(d_real_img_prob, True) * self._opt.lambda_D_prob
```
```python
def _compute_loss_D(self, estim, is_real):
        return -torch.mean(estim) if is_real else torch.mean(estim)
```
![批注 2020-05-31 114808](https://user-images.githubusercontent.com/10290923/83344159-b85e3f80-a335-11ea-8b44-0763d1865232.png)
![image](https://user-images.githubusercontent.com/10290923/83344190-03785280-a336-11ea-8561-9da63b39f82b.png)

![image](https://user-images.githubusercontent.com/10290923/83344206-230f7b00-a336-11ea-8ec0-3594608f485b.png)
